build.sh

```diff
@@ -103,39 +186,48 @@ else
   HTTP_CODE=`curl $($VERBOSE && echo "-v") $($QUIET && echo "-s")\
         http://ec2-52-88-42-163.us-west-2.compute.amazonaws.com/ \
         --write-out "%{http_code}" \
-        --form source=@source-whatwg-complete \
-        --form caniuse=@caniuse.json \
-        --form w3cbugs=@w3cbugs.csv \
-        --output .wattsi-output.zip`
+        --form source=@$HTML_TEMP/source-whatwg-complete \
+        --form caniuse=@$HTML_CACHE/caniuse.json \
+        --form w3cbugs=@$HTML_CACHE/w3cbugs.csv \
+        --output $HTML_TEMP/wattsi-output.zip`
 
   if [ "$HTTP_CODE" != "200" ]; then
-      cat .wattsi-output.zip
-      rm -f .wattsi-output.zip
+      cat $HTML_TEMP/wattsi-output.zip
+      rm -f $HTML_TEMP/wattsi-output.zip
       exit 22
   fi
 
-  unzip $($VERBOSE && echo "-v" || echo "-qq") .wattsi-output.zip -d .wattsi-output
-  rm .wattsi-output.zip
+  unzip $($VERBOSE && echo "-v" || echo "-qq") $HTML_TEMP/wattsi-output.zip -d $HTML_TEMP/wattsi-output
+  rm $HTML_TEMP/wattsi-output.zip
 fi
 
-rm -f index
-cat .wattsi-output/index-html | perl .post-process-index-generator.pl | perl .post-process-partial-backlink-generator.pl > index;
+rm -f $HTML_OUTPUT/index
+cat $HTML_TEMP/wattsi-output/index-html | perl .post-process-index-generator.pl | perl .post-process-partial-backlink-generator.pl > $HTML_OUTPUT/index;
 
 # multipage setup
-ln -s ../images .wattsi-output/multipage-html/
-ln -s ../link-fixup.js .wattsi-output/multipage-html/
-ln -s ../entities.json .wattsi-output/multipage-html/
+ln -s ../images $HTML_TEMP/wattsi-output/multipage-html/
```

Comment:
Note for the future: we should just make these 301s and fix any links, instead of creating symbolic links in the output/on the server.